server/actions_apis.go

```diff
@@ -66,17 +66,25 @@ func (s *RegistryServer) DeleteApi(ctx context.Context, req *rpc.DeleteApiReques
 		return nil, unavailableError(err)
 	}
 	defer s.releaseStorageClient(client)
-	// Validate name and create dummy api (we just need the ID fields).
+
```

Comment:
What's your criteria for deleting comments? This seems like an unintuitive use of NewApiFromResourceName() that I thought was worth explaining.

Reply:
It wasn't intentionally removed, I should pay more attention to my diffs. Fixed.